frontend/src/bundles/common/components/create-goal-form/enums/goal-activity.enum.ts

```diff
@@ -0,0 +1,15 @@
+import { capitalizeString } from '~/bundles/common/helpers/helpers.js';
+import { Activity } from '~/bundles/goals/enums/enums.js';
+
+type ActivityType = {
+    label: string;
+    value: string;
+};
+
+const goalActivity: ActivityType[] = [
+    { label: capitalizeString(Activity.CYCLING), value: Activity.CYCLING },
+    { label: capitalizeString(Activity.RUNNING), value: Activity.RUNNING },
+    { label: capitalizeString(Activity.WALKING), value: Activity.WALKING },
+];
```

Comment:
I think since we already have select option type in select component you should use it.